modules/statics/src/networks.ts

```diff
@@ -633,6 +633,18 @@ class SeiTestnet extends Testnet implements AccountNetwork {
   explorerUrl = 'https://sei.explorers.guru/transaction/';
 }
 
+class INJ extends Mainnet implements AccountNetwork {
```

Comment:
```suggestion
class Injective extends Mainnet implements AccountNetwork {
```

Let's maintain the convention followed for other coins. I think the convention is that whatever is the name of the network, is the class name as well